### ISSUE TITLE
Heatmaps with seaborn on ranks

### ISSUE THREAD

**Issue by isisannelie** [knightlab-analyses/multiomic-cooccurrences#14]
Hi! 

I am currently working on an IBD cohort and I have used mmvec to investigate the microbe-metabolite co-occurences. To validate the findings I want to produce a heatmap that shows the interactions between _R. gnavus_ and some _Alistipes_ species and caprylic acid and a few LCFA's. For my analysis I am mainly working in jupyer notebooks, so I followed the notebooks on here, like this:

```
ranks = pd.read_csv(ranks_fn, sep = "\t", index_col = 0)
gnavus = list(taxa_diff.index[['gnavus' in x for x in taxa_diff.index]])
alistipes = list(taxa_diff.index[['Alistipes' in x for x in taxa_diff.index]])

val_mic = gnavus + alistipes

LCFAs = list(compound_names.index[[('arachidonic acid' in x) or ('eicosatrienoic acid' in x) or ('docosapentaenoic acid' in x) or ('adrenic acid' in x) for x in compound_names.values]])
other_compounds = list(compound_names.index[[('caprylic acid' in x) or ('octanoic acid' in x) for x in compound_names.values]])

val_comp = LCFAs + other_compounds

ranks_val = ranks.loc[val_comp, val_mic]
ranks_val = ranks_val.transpose()
ranks_val.index = list(map(lambda x: x.strip().split("|")[-1], val_mic))
ranks_val.columns = met_diff_md.loc[val_comp, "Compound"].values
```

When I'm plotting the heatmap without any additional normalization, I will get the following plot:
`sns.clustermap(ranks_val, cmap = 'seismic', figsize = (10,10))`

![afbeelding](https://user-images.githubusercontent.com/60511351/80868992-08e26e80-8c9e-11ea-968c-1d0e5e8d1306.png)

Which doesn't really show any of the earlier found interactions as obvious (caprylic acid inhibits R. gnavus, but is positively associated with several Alistipes and DPA was found positively associated with R. gnavus).

However, when I use a z_score applied over the columns (metabolites) I will get the expected associations:

`sns.clustermap(ranks_val, z_score = 1, cmap = 'seismic', figsize = (10,10))`
![afbeelding](https://user-images.githubusercontent.com/60511351/80869079-8b6b2e00-8c9e-11ea-8c0f-cb627d15d548.png)

Beneath are the ranks shown for caprylic acid.
![afbeelding](https://user-images.githubusercontent.com/60511351/80869124-c2414400-8c9e-11ea-8135-baddbfb7e3f8.png)

So basically is it correct to use a z-score applied on the metabolites here? I got a bit confused working with the mmvec ranks at this point and I am not exactly sure how to handle them anymore. Any help would be highly appreciated :)

**Comment by mortonjt** [comment]
@isisannelie glad to hear that you are getting useful insights from mmvec!

The z-score should should be ok - we are doing a column centering anyways on rank-heatmap that should correct the issues that you are seeing (see the [mmvec readme](https://github.com/biocore/mmvec)).

If there are more questions, I'd recommend to post on the main mmvec github repo: https://github.com/biocore/mmvec  (so that we can keep everything in one place).

**Comment by mortonjt** [comment]
Hi @isisannelie I'm going close this issue. Feel free to follow up if there are lingering questions
